examples/I/hashmap.c

```diff
@@ -0,0 +1,132 @@
+#include <string.h>
+
+#include "hashmap.h"
+
+struct hashmap_t
```

Comment:
Do we have some code convention for our examples? The naming is different in other tutorials, e.g. 

https://github.com/pmemhackathon/2021-04-21/blob/master/examples/D/freq_pmem.c

Please note that the "_t" appendinx is not used. Also, I think that 1. Common convention is different 2. "_t" is a reserved POSIX suffix - I know that people use it often anyways and I don't have anything against it, but maybe it would be better to avoid it in code that will "face customers" and work as a reference for them?

Reply:
I see that each example uses slightly different coding style - I will ask Andy about that

Reply:
?

Reply:
done